RocketChat/Rocket.Chat.ReactNative
# [IMPROVE] Added support for thumbnails of videos (#2746)

* Added expo-video-thumbnails
* Added Fast-image for caching the images
* Fix touch feedback design



## Proposed changes

Modified video containers for thumbnail support. Using expo-video-thumbnail and fast-image caching, with the acceptance of this PR the user will be able to preview the video thumbnails. The thumbnails are being loaded asynchronously which won't affect the loading time for messages. Even the thumbnail can take a while to load , the user will still able to play the videos and scroll through the messages. 

## Issue(s)	
closes #2746

## How to test or reproduce

To test the feature, we can send or receive video messages and see the thumbnails 

## Screenshots
![pr-react-native](https://user-images.githubusercontent.com/69245931/227267404-f5bb4851-1f67-47d2-a469-74155c4a1fc7.gif)

## Types of changes




## Checklist



## Further comments
